### ISSUE TITLE
Image is blank on >1.7.3; Firmware 1.6.202

### ISSUE THREAD

**Issue by Mr-HaleYa** [romedtino/simple-wyze-vac#56]
**Describe the bug**
When I update to any version greater than 1.7.3 the latest image lacks all the details of the last run. it just shows the basic map

**To Reproduce**
1. Go to 1.7.3 and the image is this
![Home Vacuum_last_map (1)](https://user-images.githubusercontent.com/42821995/232343479-139e50a3-5f11-4c5b-9df1-754882c65e45.jpg)
2. Go to >1.7.3 and the image is this
![Home Vacuum_last_map](https://user-images.githubusercontent.com/42821995/232343505-251ea240-3406-471c-b95f-0a043d6b80e2.jpg)

**Simple Wyze Vac Version**
- I've tried 1.7.3 (works), 1.8, 1.8.1, 1.8.2 

**Wyze Vacuum Firmware Version**
- 1.6.202

**Home Assistant Version**
- Home Assistant 2023.4.4
- Supervisor 2023.04.0
- Operating System 9.5
- Frontend 20230411.1 - latest

**Additional context**
I've completely uninstalled and reinstalled the integration and hacks download twice restarting in between each action and delegated all references to simple-wyze-vac in the custom-components and www before reinstalling. I'm using 2FA. I've been using your integration since 1.1.2.
I've also tried adding my second floor and that did nothing so I removed it and still nothing

**Comment by romedtino** [comment]
Your map issue  should be resolved in this release - https://github.com/romedtino/simple-wyze-vac/releases/tag/1.8.3

Just for background-

The reason why the map no longer has the sweep route is because the older Simple Wyze Vac version would pull the map image from the latest sweep record available. This map would contain the house map _and_ and sweep route. When I updated to the latest version of `wyze-sdk`, I took advantage of the new `get_maps` function which _can find_ multiple maps if you have different floors but it doesn't have the sweep route details. I do like seeing the sweep details so for now what it does is:

If there is a sweep record, use the map provided by that. If there is not, look for the active map in the available map list.
